pages/index.js

```diff
@@ -8,6 +8,7 @@ import PersonDetail from '../components/user-info';
 import Filter from '../components/filter';
 import { Footer } from '../components/footer';
 import styles from '../styles/Home.module.css';
+import TransactionChart from '../components/transaction-chart/index.js';
```

Comment:
We can include just the component folder and not the index.js in it, following the convention from the existing codebase 😁

Reply:
Thanks for the review Swaraj, I have made the changes accordingly :)